games-action/openjazz/patches/openjazz-20180522.patchset

```diff
@@ -0,0 +1,24 @@
+From 2f1a970b77764088a86b5b611ef563ecf1771fed Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 5 Jan 2019 21:00:55 +0100
+Subject: fix build with missing sys/select.h (fd_set)
+
+
+diff --git a/src/io/network.cpp b/src/io/network.cpp
+index 7a888f7..76047cd 100644
+--- a/src/io/network.cpp
++++ b/src/io/network.cpp
+@@ -59,7 +59,9 @@
+ #elif defined USE_SDL_NET
+ 	#include <arpa/inet.h>
+ #endif
+-
++#ifdef __HAIKU__
++	#include <sys/select.h>
```

Comment:
Any idea why it worked before without this?
Can also upstream this patch.

Reply:
I don't know to be honest, just came across this when I was looking over some of the older PR's here, was thinking about upstreaming it but this would even be better, thanks! :+1:

Reply:
If you have upstreamed the patches could you give a notice @carstene1ns ? I'll do a git checkout to look into it before the next release with them included then :)

Reply:
We do have [`<sys/select.h>`](http://xref.plausible.coop/source/xref/haiku/headers/posix/sys/select.h). What error did you get with that `#include`?

Reply:
Often other os will include sys/select.h from another .h file, where we don't. So in Haiku it needs to be included explicitly if you use it.

@alaviss: this is a #ifdef not #ifndef!